guide/plugins_wholebody.ipynb

```diff
@@ -0,0 +1,178 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import IPython\n",
+    "IPython.display.Image('wholebody/soccer.jpeg.predictions.jpeg')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eb3a3193",
+   "metadata": {},
+   "source": [
+    "Image credit: [Photo](https://de.wikipedia.org/wiki/Kamil_Vacek#/media/Datei:Kamil_Vacek_20200627.jpg) by [Lokomotive74](https://commons.wikimedia.org/wiki/User:Lokomotive74) which is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2a6621dd",
+   "metadata": {},
+   "source": [
+    "## Visualization of the additional keypoints\n",
+    "Original MS COCO skeleton / COCO WholeBody skeleton"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "6dfbbb5b",
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "TypeError",
```

Comment:
There is an error here.

Reply:
Thank you! I incorporated all the comments in my last commits.